helm-deployment/dex-subchart/README.md

```diff
@@ -10,7 +10,14 @@ You have to set all values specify in this chapter to properly run this chart
 ```
 
 Inference Model Manager requires TSL for internal traffic. We recommend to use for this purpose our script ```generate-dex-certs.sh``` located in ```certs``` directory.
-Before running the script mentioned above, set the environment variable ```DEX_NAMESPACE```
+We also provide script to generate self-signed certificates for dex for external traffic. It can be found in following path: ```inference-model-manager/helm-deployment/dex-subchart/certs/generate-ing-dex-certs.sh```
+Before running scripts mentioned above, set the environment variable ```DEX_NAMESPACE, DEX_DOMAIN_NAME, ISSUER```
+Format of above variables should fit following patterns:
+```
+export ISSUER=https://dex.<your domain>:443/dex # change 443 port if using kubernetes node port instead of load balancer
+export DEX_NAMESPACE=dex # change value of this variable only if you understand consequences
```

Comment:
Are there some consequences? Why not just recommend the default namespace name. If the name is hardcoded somewhere perhaps it should be just explained what are the dependencies.